src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
```

Comment:
1. Good use of `@Service` annotation to indicate that the `CalculatorService` class is a Spring-managed service.
2. Methods `add`, `subtract`, `multiply`, and `divide` are straightforward and perform the respective mathematical operations.
3. In the `divide` method, there is a check for division by zero to prevent an `IllegalArgumentException` from being thrown.
4. It might be beneficial to consider handling division by zero with more specific exception types, such as `ArithmeticException`, which is more suited for mathematical operations.
5. The code does not currently have any unit tests written for the methods. It would be helpful to add unit tests to ensure the correctness of the mathematical operations in the `CalculatorService` class.